docs/hub/_toctree.yml

```diff
@@ -63,6 +63,8 @@
     sections:
       - local: adapters
         title: Adapters
+      - local: agents
+        title: Agents on Hub
```

Comment:
i would put it in Other, maybe between Paper pages and Search:

<img width="352" alt="image" src="https://github.com/user-attachments/assets/36cf137c-8e42-4930-bfe6-538c37a3f150" />